vortex-ffi/cinclude/vortex.h

```diff
@@ -159,12 +164,26 @@ typedef struct vx_file_scan_options {
    * Number of columns in `projection`.
    */
   int projection_len;
+  /**
+   * Serialized expressions for pushdown
+   */
   const char *filter_expression;
+  /**
+   * The len in bytes of the filter expression
+   */
   int filter_expression_len;
   /**
    * Splits the file into chunks of this size, if zero then we use the write layout.
    */
   int split_by_row_count;
+  /**
+   * First row of a range to scan.
+   */
+  unsigned long row_range_start;
+  /**
+   * Last row of a range to scan.
+   */
+  unsigned long row_range_end;
```

Comment:
Are all these opts compatible?

Reply:
Talked with Nick about this, landing on that we will probably change the options FFI API to providing a setter for each option.

Reply:
With an error in the setter if you do it wrong?

Reply:
What is the error you want to guard against? If the split by value is sufficiently large, no additional splits happen. 

That said, I don't think verifying the combinations of these is up to the FFI layer but the respective stream/scanbuilder builders. E.g. should then be checked in scan/mod.rs.